src/Notifications/Web/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
```

Comment:
@filipeotero if we are using WebView2 is this line still needed?

Reply:
not necessary, removed it. Thanks!